resource.qrc

```diff
@@ -1,8 +1,7 @@
 <RCC>
     <qresource prefix="/">
-        <file>database/slika.jpg</file>
-        <file>database/slika2.jpg</file>
-        <file>database/purple-glitter.png</file>
-        <file>database/Purple-Confetti</file>
+        <file>database/picture.jpg</file>
+        <file>database/picture2.jpg</file>
+        <file>database/purple-confetti</file>
```

Comment:
Da li se ovo mora zvati "database"? Nekako mi nije prikladno, s obzirom na to da su slike u pitanju. Više mi je kao neki "resources" ili "assets".